### ISSUE TITLE
Aggregate Fuzzer Test failure

### ISSUE THREAD

**Issue by spershin** [facebookincubator/velox#9177]
### Description

https://app.circleci.com/pipelines/github/facebookincubator/velox/46498/workflows/bae0649b-899f-461c-875c-6964445391c2/jobs/321286

### Error Reproduction

https://app.circleci.com/pipelines/github/facebookincubator/velox/46498/workflows/bae0649b-899f-461c-875c-6964445391c2/jobs/321286

### Relevant logs

```bash
https://app.circleci.com/pipelines/github/facebookincubator/velox/46498/workflows/bae0649b-899f-461c-875c-6964445391c2/jobs/321286
```

**Comment by mbasmanova** [comment]
```
I20240320 06:15:37.650759 269128 AggregationFuzzerBase.cpp:391] Executing query plan: 
-- Aggregation[SINGLE [g0, g1] a0 := regr_r2(ROW["c0"],ROW["c1"])] -> g0:INTEGER, g1:BIGINT, a0:DOUBLE
  -- Values[1000 rows in 10 vectors] -> c0:DOUBLE, c1:DOUBLE, g0:INTEGER, g1:BIGINT
```

Looks like a duplicate of #9089

Fix is out for review: #9089

CC: @8dukongjian

**Comment by spershin** [comment]
Closing this as a duplicate.
Also the fix has just landed. https://github.com/facebookincubator/velox/issues/9089
